rendering-event-loop.md

```diff
@@ -0,0 +1,94 @@
+### Event loops
+
+HTML and script in web pages use an event loop-driven model of computation. The event loops for HTML are defined [here](https://html.spec.whatwg.org/multipage/webappapis.html#event-loop). Tasks on an event loop live in one or more [task queues](https://html.spec.whatwg.org/multipage/webappapis.html#task-queue), and each loop only has one task at a time. Event loops also have [microtask queues](https://html.spec.whatwg.org/multipage/webappapis.html#microtask), which are generally processed all-at-once at the end of each time script may run, or at the end of tasks. These are called [microtask checkpoints](https://html.spec.whatwg.org/#perform-a-microtask-checkpoint).
```

Comment:
"at the end of each time script may run" -> is this before script execution? It's a bit awkward and I'm not sure I get what it means...

"these are called microtask checkpoints" -> "these times are called microtask checkpoints"

Reply:
(as a meta comment, it'd be nice to format the md to 80 col or something similar)

Reply:
> "at the end of each time script may run" -> is this before script execution? It's a bit awkward and I'm not sure I get what it means...

Done

> "these are called microtask checkpoints" -> "these times are called microtask checkpoints"

Done



> (as a meta comment, it'd be nice to format the md to 80 col or something similar)
> 

I can't do that, because explicit newlines are treated as formatting by markdown.

Reply:
> I can't do that, because explicit newlines are treated as formatting by markdown.

Actually I can. Fixed.